assets/js/highscore.js

```diff
@@ -0,0 +1,60 @@
+//These functions will be primarily used for the creation, storing, and overall handling of the game high scores.
+//At this time, the functions will be coded under the assumption that each game will have five high scores.
+
+//Sets how many high scores will be saved
+const totalHighScores = 5;
+
+//Used for array/string translation
+var highScoresArray = getGameName(gameName);
+
+function getGameName (gameName) {
+    gameName += '-hs';
+    return gameName;
+}
+
+//The purpose of this function is to check if the player's score is a "high score"
+//The funciton should be called by whatever script will control the "game over" part of the games
+function highScoreChecker(gameScore) {
```

Comment:
Need to have a way to pass  the game name when a game calls the highscorechecker function